pino.d.ts

```diff
@@ -123,7 +123,7 @@ export interface LoggerExtras<Options = LoggerOptions> extends EventEmitter {
     /**
      * Flushes the content of the buffer when using pino.destination({ sync: false }).
      */
-    flush(): () => void;
+    flush(): void;
```

Comment:
Can you add a test for this type? We use tsd.

Reply:
Oh, I did understand you meant that, np